lib/tasks/mqtt_subscriber.rake

```diff
@@ -13,6 +13,7 @@ namespace :mqtt do
         client.subscribe({
           '$queue/device/sck/+/readings' => 2,
           '$queue/device/sck/+/hello' => 2,
+          '$queue/device/sck/+/hardware_info' => 2,
```

Comment:
@viktorsmari topic should be `info` instead as that's the one used by the firmware already
https://github.com/fablabbcn/smartcitizen-kit-20/blob/1b36eded5c6d514aa10c055b81b8c211e3622a9e/esp/src/SckESP.cpp#L352`